modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1280,6 +1302,16 @@ AUI.add(
 						var selectButtonNode = container.one('#' + instance.getInputName() + 'SelectButton');
 
 						selectButtonNode.attr('disabled', instance.get('readOnly'));
+
+						var clearButtonNode = container.one('#' + instance.getInputName() + 'ClearButton');
+
+						clearButtonNode.attr('disabled', instance.get('readOnly'));
+
+						var altNode = container.one('#' + instance.getInputName() + 'Alt');
```

Comment:
Hi @rolandpakai,

In which use case this altNode is necessary in the Documents and Media field?

Reply:
Nevermind, I figured out you need to do that because of the input Image Description. 😉